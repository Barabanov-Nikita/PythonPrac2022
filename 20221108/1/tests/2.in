a = DivStr("abcdefghijklmnopqrstuvwxyz")
print(* a // 2)
print(a % 5)
print(* a % 10 // 4)
print(a.upper() % 3)
print(* a[1:5] // 3)
print(DivStr())
print(a % 4 + DivStr() + a % 5)