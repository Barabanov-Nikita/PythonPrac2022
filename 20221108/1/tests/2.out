abcdefghijklm nopqrstuvwxyz
z
u v w x
YZ
b c d

yzz
